### PR TITLE
modified: codecademy/Frontend-Development

### PR DESCRIPTION
**Pull request by abhijeetdav24aug**
## Screenshots

image
correcting the spelling for codecademy/HTML/Frontend-Development

### CODE REVIEW

File: Frontend-Development/README.md [hunk 1]
```diff
@@ -56,12 +69,15 @@
 - [CSS Diner](https://flukeout.github.io/)
 
 ### Frontend Projects
+
 - [frontendmentor.io](https://www.frontendmentor.io/)
 - [frontendpractice.com](https://www.frontendpractice.com/)
 - [iCodeThis.com](https://www.icodethis.com/)
 
 ## Learning Javascript
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:78]: Remove the whitespace

File: Frontend-Development/README.md [hunk 2]
```diff
@@ -36,13 +46,16 @@
...
 
 ### Frontend Projects
+
 - [frontendeval.com](https://www.frontendeval.com)
 - [frontendmentor.io](https://www.frontendmentor.io/)
 - [frontendpractice.com](https://www.frontendpractice.com/)
 - [iCodeThis.com](https://www.icodethis.com/)
 
 ## Learning CSS
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:56]: Remove the whitespace

File: Frontend-Development/README.md [hunk 3]
```diff
@@ -20,12 +27,15 @@
 - [What is DNS? | How DNS works](https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/)
 
 ### Internet-CS50's understanding technology
+
 - [Resource](https://www.youtube.com/watch?v=n_KghQP86Sw)
 
 ## Learning HTML
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:34]: Remove the whitespace

File: Frontend-Development/README.md [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
 - [Frontend web developer - Mozilla Developer docs](https://developer.mozilla.org/en-US/docs/Learn/Front-end_web_developer)
 - [What is frontend development - Freecodecamp](https://www.freecodecamp.org/news/front-end-developer-what-is-front-end-development-explained-in-plain-english/)
+
 ### What is frontend web development
+
 - [Frontend Development explained - Mayuko(Youtube)](https://www.youtube.com/watch?v=qyHyFsT7Hig)
 - [Frontend web development - Wikipedia](https://en.wikipedia.org/wiki/Front-end_web_development)
 
 ## Understanding Internet
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:17]: Remove the whitespace

File: Frontend-Development/README.md [hunk 5]
```diff
@@ -1,15 +1,22 @@
 # Frontend Roadmap
+
 ## Introduction
+
```
Comment (siddhant-khisty) [inline Frontend-Development/README.md:4]: Remove the whitespace

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

File: Frontend-Development/README.md (hunk 4 above)
Comment (abhijeetdav24aug) [reply Frontend-Development/README.md:17]: sir which whitespaces you are talking about ? Can you give some examples?

**Review by siddhant-khisty — APPROVED** [review]
lgtm

**Pull request merged**